scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e
+
+echo "Adding the stackrox/helm-charts/opensource repository to Helm."
+
+helm repo add stackrox https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+
+echo "Generating stackrox-admin-password.txt"
+
+openssl rand -base64 20 | tr -d '/=+' > stackrox-admin-password.txt
+STACKROX_ADMIN_PASSWORD=`cat stackrox-admin-password.txt`
+
+echo "Installing stackrox-central-services"
+
+helm install -n stackrox --create-namespace stackrox-central-services stackrox/stackrox-central-services --set central.adminPassword.value="$STACKROX_ADMIN_PASSWORD"
+
+kubectl -n stackrox rollout status deploy/central --timeout=3m
+
+echo "Generating an init bundle with shared secrets."
+
+kubectl -n stackrox exec -it deploy/central -- roxctl --insecure-skip-tls-verify \
+  --password "${STACKROX_ADMIN_PASSWORD}" \
+  central init-bundles generate stackrox-init-bundle --output - 0>/dev/null 1> stackrox-init-bundle.yaml 2>/dev/null
```

Comment:
Why not simply 

```suggestion
kubectl -n stackrox exec -it deploy/central -- roxctl --insecure-skip-tls-verify \
  --password "${STACKROX_ADMIN_PASSWORD}" \
  central init-bundles generate stackrox-init-bundle --output stackrox-init-bundle.yaml
```

If any errors happen, it would be good that the users see them.

Reply:
The reason I've piped those outputs to /dev/null is that the StdErr stream would end up within my stackrox-init-bundle.yaml file and lease to parsing errors. I was unable to figure out how to pipe StdErr to the terminal instead, do you know how to do this properly?

Reply:
Okay, this was the main reason I requested changes, so let me investigate it closer. I will follow-up on this

Reply:
I am testing this now and I think that there is a simpler way of achieving this. Will follow up on this

Reply:
Ok, I think the solution with port-forward + `--output` to file will be much more robust against the potential warnings. This is the way we use it in our scripts in stackrox. 

```
kubectl -n stackrox port-forward deploy/central 8000:8443 > /dev/null 2>&1 &

# we might need a sleep 1, or some active polling here 

# optionally, test connection to the cluster
roxctl central --insecure-skip-tls-verify --password "${STACKROX_ADMIN_PASSWORD}" --endpoint localhost:8000 init-bundles list

roxctl central \
  --insecure-skip-tls-verify \
  --password "${STACKROX_ADMIN_PASSWORD}" \
  --endpoint localhost:8000 \
  init-bundles generate stackrox-init-bundle \
  --output stackrox-init-bundle.yaml
```

Reply:
@jschnath let's correct this part ☝️ and maybe give it a test on Linux and this will be good to go

Reply:
This works on my machine but not on my Linux VM since I hadn't installed roxctl on it. This is also the reason why we had opted to use roxctl from within ´kubectl -n stackrox exec -it deploy/central --´ so far since it avoids adding the additional requirement of installing roxctl.

I would like to still avoid this if possible, at least personally the tradeoff for not showing errors while generating init bundles seems worth the reduced installation requirements. If there is no way to pipe 0> and/or 2> into the terminal running ´kubectl -n stackrox exec -it deploy/central --´ then perhaps it would make sense to pipe them into a log file to check if something went wrong?

Reply:
Ok, I understand that this does not work without `roxctl`. I feel that the solution with so many redirects can turn out to be really unreliable and I would like to avoid it all cost. Can we maybe somehow `curl` into central and get the bundle?

Reply:
Another alternative would be to execute this in the k8s container and later cat it, sth like this (untested):

```sh
kubectl -n stackrox exec -it deploy/central -- \
   bash -c "roxctl --insecure-skip-tls-verify \
  --password \""${STACKROX_ADMIN_PASSWORD}"\" \
  central init-bundles generate stackrox-init-bundle \
  --output stackrox-init-bundle.yaml > /dev/null 2>&1 && cat stackrox-init-bundle.yaml"
```

Reply:
It turns out that removing the -it option made all the redirects unnecessary. Running 
```
kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
  --password "${STACKROX_ADMIN_PASSWORD}" \
  central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
```
  
 It now works as expected